src/lib/Microsoft.Health.Extensions.Fhir.R4/FhirServiceValidator.cs

```diff
@@ -0,0 +1,32 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Hl7.Fhir.Rest;
+using Microsoft.Health.Extensions.Fhir.Telemetry.Exceptions;
+using Microsoft.Health.Logging.Telemetry;
+
+namespace Microsoft.Health.Extensions.Fhir
+{
+    public static class FhirServiceValidator
+    {
+        public static bool ValidateFhirService(FhirClient client, ITelemetryLogger logger)
```

Comment:
I don't see us utilizing this bool returned.

Reply:
The bool's currently used only in the unit test for this function. I'm also thinking that this function can be used elsewhere in the future.